pkgs/applications/misc/easystroke/default.nix

```diff
@@ -0,0 +1,55 @@
+{ stdenv, fetchFromGitHub, fetchpatch, pkgconfig, boost, dbus_glib, gnome3,
```

Comment:
Why did you specify `gcc7` there (& within nativeBuildInputs)? Usually sticking with just using whatever `stdenv` provides is what you want.